### ISSUE TITLE
Brief description of your problem

### ISSUE THREAD

**Issue by EmeraldSplash14** [PixelExperience/android-issues#2943]
## Build date

```
PixelExperience_Plus_taimen-12.1-20220615-1912-OFFICIAL
```

## Expected Behavior

```
The Voice Access accessibility app should be able to listen for voice commands and allow you to control your phone hands-free.
```

## Current Behavior

```
The Voice Access app will turn on but will not listen for any voice commands. The blue bubble in the top left corner is stuck in an animation loop as if it were constantly restarting, and no live transcript of what you are saying appears at the top of the screen. Due to this, I can no longer control my phone with my voice.
```

## Possible Solution

```
Unsure. I tried restarting the phone again, uninstalling and reinstalling the app, and made sure all microphone permissions were on, but nothing has made the app work again.
```

## Steps to Reproduce

```
1. Install the Voice Access app from the Google Play Store and go to Settings>Accessibility>Voice Access and toggle on "Use Voice Access" to use the app.
2. After following the previous step and completing the tutorial for Voice Access, the app should be working properly at first. Now restart the phone.
3. After restarting the phone, turn on Voice Access. 
4. The app should no longer be working as expected.
```

/device taimen (https://download.pixelexperience.org/taimen)
/version twelve_plus

**Comment by pixelexperiencebot** [comment]
Validation failed, this issue will be closed for not complying with the established standards.

Errors:
- Invalid device: taimen (https://download.pixel
